spring-kafka/src/main/java/org/springframework/kafka/annotation/EnableKafkaRetryTopic.java

```diff
@@ -87,5 +82,6 @@
 @Target(ElementType.TYPE)
 @Documented
 @Import(RetryTopicConfigurationSupport.class)
-public @interface EnableRetryTopic {
+@EnableKafka
```

Comment:
I don't think this has to be there. Or at least it has to be documented properly: 
"For convenience the `@EnableKafkaRetryTopic` is marked also with the `@EnableKafka` since former cannot work without secondary"
(Or some other proper English language 😄 )

Reply:
No, there is no harm: the `@EnableKafka` does a decent work to not duplicate beans. See its target `KafkaBootstrapConfiguration`.
I'm not sure that retry topics should be a default feature.
Therefore keep that `@EnableKafkaRetryTopic` is the way to go from now on.

Reply:
Well, if we're introducing a new annotation then, and probably 99% of the users will be using it in a `Spring Boot` application anyway, I'd stick with the simpler `@EnableRetryTopic` annotation that should be enough for them, and just add a note in the documentation that if users are not in a `Spring Boot` app they should also add `@EnableKafka`.

But no worries, I'll keep it as is and we can revise this down the road if needed.

Thanks.

Reply:
Just as a thought, of course we can look into this later, but what if for 3.0 we added this class to `@EnableKafka`, but also added a custom `@Condition` that would perform that annotation / bean lookup we had in the other PR, to only add this beans it if there's at least one `RetryTopicConfiguration` or `@RetryableTopic` present?

This way we'd keep the automatic feature enabling users currently have, while not encumbering other users with this if they don't use it.

Thanks.

Reply:
Well, that's not how other `@Enable...`  features work. So, my vote for "No": if you are going to use `@RetryableTopic`, then add respective `@Enable...`.
The condition could be considered in Spring Boot which indeed has some opinion.
The framework by itself has to be as straightforward as possible. 
In the end it is just a library with minimal opinion 😉

Reply:
Sure, makes sense. As you said in another post, eventually it'll be a good thing to better integrate this with `Spring Boot`'s auto configuration in the future. But since this approach won't be available for `Spring Boot` prior to 3.0, we don't really need to worry about it now.

Thanks.

Reply:
I still want to see some mentioning of this meta-annotation in the Javadocs of this `EnableKafkaRetryTopic` class.